web/styles/rendered_markdown.css

```diff
@@ -569,14 +590,18 @@
     .message_embed {
         display: block;
         position: relative;
-        margin: 5px 0;
+        margin: 0 0 5px;
         border: none;
         border-left: 3px solid hsl(0deg 0% 93%);
         height: 80px;
         padding: 5px;
         z-index: 1;
         text-shadow: hsl(0deg 0% 0% / 1%) 0 0 1px;
 
+        &:last-child {
+            margin-bottom: 0;
+        }
```

Comment:
I wonder if it'd be cleaner to do this style of `last-child` override for the other elements, just to have the CSS for each element be more self-localized?

Reply:
Possibly, although I think @andersk's [points below](https://github.com/zulip/zulip/pull/29756#pullrequestreview-2004942043) deserve consideration and discussion before making that change, if we go this broad route.